### ISSUE TITLE
Report spam

### ISSUE THREAD

**Issue by ghost** [bellroy/predictionbook#86]
Something like http://predictionbook.com/predictions/177663 rings alarm bells as possible spam.

If predictionbook gets popular enough expect 99%+ of the claims to ads for viagra.

**Comment by jaysonvirissimo** [comment]
Yeah, see [here](https://predictionbook.com/predictions/209295) for a recent example:
<img width="893" alt="Screenshot 2023-01-08 at 6 29 11 PM" src="https://user-images.githubusercontent.com/1292454/211228919-a0b81210-d320-4252-ab9c-cb80d5449351.png">
